tool/site/client/src/components/dispense/createDispenseNotification.ts

```diff
@@ -195,9 +197,10 @@ function createDispensedQuantity(
 
 function createMessageHeader(
   prescriptionOrderMessageHeader: fhir.MessageHeader,
-  focusResourceIds: Array<string>
+  focusResourceIds: Array<string>,
+  replacementOfId: string | null,
```

Comment:
replacementOfId is pulled as a URL param and passed in, if it isn't in the URL param it would be `null` rather than `undefined`. We could get away with using `replacementOfId?` but I prefer to match the types as close as possible.